Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PYTHON_VERSION=3.10
+FROM python:${PYTHON_VERSION}-slim AS base
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user
```

Comment:
Consider using a more secure method for creating users, such as specifying a UID and GID, to avoid potential conflicts with existing users on the host system.